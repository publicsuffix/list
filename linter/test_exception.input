// test:
// - valid exception
// - invalid exceptions
// - same exception twice
// - exception without wildcard
// - exception with prevailing '*' rule (!localhost)

// ===BEGIN ICANN DOMAINS===

// valid
*.example.com
!www.example.com
!localhost
c.example.com

// invalid
!.example.com
w!w.example.com
!www.example.com
!a.b.example.com
!a.c.example.com

// ===END ICANN DOMAINS===
