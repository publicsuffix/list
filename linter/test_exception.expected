17: error: Leading/trailing or multiple dot: '!.example.com'
18: error: Illegal character: 'w!w.example.com'
19: error: Found doublette/ambiguity (previous line was 12): '!www.example.com'
20: error: Exception without previous wildcard: '!a.b.example.com'
21: error: Exception without previous wildcard: '!a.c.example.com'
23: warning: No PRIVATE section found
